docs/manual/appendix/JVMMemoryOnDev.md

```diff
@@ -0,0 +1,32 @@
+# Increase Memory in SBT and Maven
+
+Lagom in dev mode starts all your services plus some internal Lagom services and in a single JVM. This can cause `OutOfMemoryError`s depending on your JVM setup. It is possible to start maven and sbt with increased memory.
+
+We recommend you increase the Maximum Metaspace size and the Thread Stack size. These values can be set using `-Xss2M -XX:MaxMetaspaceSize=1024M`. The exact values may depend on your hardware and your code base.
+
+
+## MAVEN
+
+You can start maven with extra memory using `MAVEN_OPTS` environment variable.
+
+```bash
+$ MAVEN_OPTS="-Xms512M -Xmx1024M -Xss2M -XX:MaxMetaspaceSize=1024M" mvn lagom:runAll
+```
+
+In this example we're setting an initial JVM Heap of 512Mb, a maximum Heap of 1024M, a thread stack of 2M and maximum Metaspace size of 2014M. 
+
+Stating the `MAVEN_OPTS` on every invocation is error prone and exporting it globally might not be possible if you need different settings for different projects. You may want to use [direnv](http://direnv.net/) to setup the environment variables per project.
+
+## SBT
+
+Sbt let's you list the JVM options you need to run your project on a file named `.jvmopts` in the root of your project. 
```

Comment:
Also, I think this is only true as of sbt 0.13.13, which was released very recently, so it's probably worth mentioning.